core/middleware.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+class PageViewLoggerMiddleware:
```

Comment:
I recommend some tests for this middleware. Here is an example, more complex than what you need here (I think), on how to test them:

https://github.com/adamchainz/django-htmx/blob/main/tests/test_middleware.py

Reply:
btw this is one of the persons that run the django meetup today!